build_runner/test/asset_graph/graph_test.dart

```diff
@@ -86,7 +86,8 @@ void main() {
                 makeAssetId(), md5.convert(UTF8.encode('test')));
 
             var generatedNode = new GeneratedAssetNode(0, node.id, g % 2 == 1,
-                g % 2 == 0, makeAssetId(), builderOptionsNode.id);
+                g % 2 == 0, makeAssetId(), builderOptionsNode.id,
+                isHidden: false);
```

Comment:
lets make this alternate back and forth like the others, maybe `g % 3 == 0`?

Reply:
Done